<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InventoryFrame</class>
 <widget class="QFrame" name="InventoryFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>921</width>
    <height>655</height>
   </rect>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string notr="true">Medication search</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="nameLabel">
          <property name="toolTip">
           <string comment="MEDICATION/BARCODE SEARCH"/>
          </property>
          <property name="text">
           <string>MEDICATION / BARCODE</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="nameField">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string comment="MEDICATION/BARCODE SEARCH"/>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>60</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="lotLabel">
          <property name="toolTip">
           <string comment="LOT SEARCH"/>
          </property>
          <property name="text">
           <string comment="LOT LABEL SEARCH">LOT NUMBER</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lotField">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string comment="LOT SEARCH"/>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="stockCheckbox">
          <property name="text">
           <string notr="true">In stock</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>60</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="activeCheckbox">
          <property name="text">
           <string notr="true">Active</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>60</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="notExpiredCheckbox">
          <property name="text">
           <string notr="true">Not Expired</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="resetButton">
       <property name="text">
        <string notr="true">Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="text">
        <string notr="true">Search</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>60</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QTableWidget" name="resultTable">
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="columnCount">
      <number>9</number>
     </property>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>100</number>
     </attribute>
     <column>
      <property name="text">
       <string>MEDICATION</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>FORM</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>STRENGTH</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>UNIT SIZE</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>EXPIRATION</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>LOT NUMBER</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>UNITS ORDERED</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>UNITS LEFT</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>WRITE OFF</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
  <action name="writeOffAction">
   <property name="text">
    <string notr="true">Write Off...</string>
   </property>
   <property name="toolTip">
    <string>Write off a specified amount of stock.</string>
   </property>
  </action>
  <action name="printBarcodeAction">
   <property name="text">
    <string notr="true">Print Barcode</string>
   </property>
   <property name="toolTip">
    <string>Print a barcode</string>
   </property>
  </action>
  <action name="toggleAction">
   <property name="text">
    <string notr="true">Toggle Active</string>
   </property>
   <property name="toolTip">
    <string>Toggle active/inactive</string>
   </property>
  </action>
  <action name="increaseAction">
   <property name="text">
    <string notr="true">Increase Inventory</string>
   </property>
   <property name="toolTip">
    <string>Increase the inventory received and remaining</string>
   </property>
  </action>
  <action name="viewAction">
   <property name="text">
    <string notr="true">View Inventory</string>
   </property>
   <property name="toolTip">
    <string>View inventory record information</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>nameField</tabstop>
  <tabstop>lotField</tabstop>
  <tabstop>stockCheckbox</tabstop>
  <tabstop>activeCheckbox</tabstop>
  <tabstop>notExpiredCheckbox</tabstop>
  <tabstop>searchButton</tabstop>
  <tabstop>resetButton</tabstop>
  <tabstop>resultTable</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>nameField</sender>
   <signal>returnPressed()</signal>
   <receiver>searchButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lotField</sender>
   <signal>returnPressed()</signal>
   <receiver>searchButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>447</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetButton</sender>
   <signal>clicked()</signal>
   <receiver>nameField</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetButton</sender>
   <signal>clicked()</signal>
   <receiver>lotField</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>447</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
